txredisapi.py

```diff
@@ -2424,71 +2427,71 @@ def makeShardedConnection(hosts, dbid, poolsize, reconnect, isLazy,
 
 
 def Connection(host="localhost", port=6379, dbid=None, reconnect=True,
-               charset="utf-8", password=None,
+               charset="utf-8", password=None, use_ssl=False,
                connectTimeout=None, replyTimeout=None, convertNumbers=True):
     return makeConnection(host, port, dbid, 1, reconnect, False,
-                          charset, password, connectTimeout, replyTimeout,
+                          charset, password, ssl, connectTimeout, replyTimeout,
```

Comment:
I guess this is a typo here

```suggestion
                          charset, password, use_ssl, connectTimeout, replyTimeout,
```